.circleci/config.yml

```diff
@@ -566,7 +566,7 @@ jobs:
   node-jest:
     <<: *node-plugin-base
     docker:
-      - image: node:8
+      - image: node:10
```

Comment:
we were compatible by luck with `jest-environment-node` since they both require `>10`: https://github.com/facebook/jest/blob/master/packages/jest-environment-jsdom/package.json#L30 
https://github.com/facebook/jest/blob/master/packages/jest-environment-node/package.json#L28